org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/XMLAssert.java

```diff
@@ -766,6 +766,33 @@ public static void testDocumentSymbolsFor(XMLLanguageService xmlLanguageService,
 
 	}
 
+	public static void testDocumentSymbolsNumber(String xml, String fileURI, XMLSymbolSettings symbolSettings,

```

Comment:
XMLAssert is a kind of framework to test the XML language services features.

As this XMLAssert class is used for any extension, I would like to provide only services test which are a lot of used (ex : test for completion, hover, validation, etc). 

I'm not sure that a lot extension will do some test with testDocumentSymbolsNumber. So please move those methods in XMLDocumentSymbolsTest.java. 

We could change that according user issues.